inc/cleantalk-public-integrations.php

```diff
@@ -3324,7 +3262,11 @@ function apbct_form__elementor_pro__addField($content)
 {
     global $apbct;
 
-    if ( $apbct->settings['trusted_and_affiliate__under_forms'] === '1' && strpos($content, '</form>') !== false ) {
+    $search = '</form>';
+    $replace = ct_add_honeypot_field('elementor_form') . $search;
```

Comment:
Please, check if the option `Add a honeypot field` is enabled.

Reply:
Option checks inside ct_add_honeypot_field() https://github.com/CleanTalk/wordpress-antispam/blob/master/inc/cleantalk-public.php#L699